### ISSUE TITLE
Castle Toolbar Accessibility: Ensures buttons have discernible text

### ISSUE THREAD

**Issue by rnunez80** [castlecms/castle.cms#8]
wcag2a
wcag412
section508
section508.22.a

**Target:**
#edit-bar > .castle-toolbar-container > .castle-toolbar-container-top > .castle-toolbar-buttons > .castle-btn-dropdown.castle-btn-dropdown-cog > .icon-cog.plone-btn.plone-btn-default

**HTML:**
`<button class="icon-cog plone-btn plone-btn-default" data-reactid=".0.0.2.1.0"></button>`

**Summary:**
_Fix any of the following:_
Element has a value attribute and the value attribute is empty
Element has no value attribute or the value attribute is empty
Element does not have inner text that is visible to screen readers
aria-label attribute does not exist or is empty
aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty or not visible
Element's default semantics were not overridden with role="presentation"
Element's default semantics were not overridden with role="none"

_Fix all of the following:_
Element is in tab order and does not have accessible text

**Comment by RobZoneNet** [comment]
Put a title in there for screen readers
